### PR TITLE
Webbrowser plugin for mac

### PR DESCRIPTION
**Pull request by tofutim**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)
How about WebBrowser plugin for Mac? You know you want to...

### :arrow_heading_down: What is the current behavior?
No WebBrowser for Mac

### :new: What is the new behavior (if this is a feature change)?
WebBrowser for Mac!

### :boom: Does this PR introduce a breaking change?
Nope

### :bug: Recommendations for testing
Umm, try the plugin? How are the others tested?

### :thinking: Checklist before submitting

- [ ] All projects build
- [ ] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [ ] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contribute/mvvmcross-docs-style-guide))
- [ ] Nuspec files were updated (when applicable)
- [ ] Rebased onto current develop

### CODE REVIEW

**Comment by tofutim** [comment]
I need help on output path and anything else needed for nuget...

**Comment by tofutim** [comment]
I think I got it, someone check it.

File: nuspec/MvvmCross.Plugin.WebBrowser.nuspec [hunk 1]
```diff
@@ -45,5 +45,8 @@ This package contains the 'WebBrowser' plugin for MvvmCross</description>
 			
 		<!-- mac -->
 		<file src="..\bin\Release\Mvx\Portable\MvvmCross.Plugins.WebBrowser.*" target="lib\Xamarin.Mac20" />
+                <file src="..\bin\Release\Mvx\Mac\MvvmCross.Plugins.WebBrowser.Mac.*" target="lib\Xamarin.Mac20" />
+                <file src="MacBootstrapContent\WebBrowserPluginBootstrap.cs.pp"
```
Comment (Cheesebaron) [inline nuspec/MvvmCross.Plugin.WebBrowser.nuspec]: This doesn't exist

File: TestProjects/Mac-Support/Collections/Collections.Core/Collections.Core.csproj [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (martijn00) [inline TestProjects/Mac-Support/Collections/Collections.Core/Collections.Core.csproj]: Why is this project added?

File: TestProjects/Mac-Support/Collections/Collections.Core/Class1.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace Collections.Core
```
Comment (martijn00) [inline TestProjects/Mac-Support/Collections/Collections.Core/Class1.cs]: This class can be removed?

**Comment by martijn00** [comment]
Can you do a rebase and fix the issue's?

**Pull request merged**